modules/ocf_desktop/manifests/suspend.pp

```diff
@@ -10,8 +10,19 @@
     '/etc/acpi/events/powerbtn-acpi-support':
       source  => 'puppet:///modules/ocf_desktop/suspend/powerbtn-acpi-support',
       require => Package['acpi-support-base'];
+
+    # place files to handle daisy chain sleep issue
+    '/usr/lib/pm-utils/sleep.d/999fix-daisy':
+      mode   => '0755',
+      source => 'puppet:///modules/ocf_desktop/suspend/fix-daisy';
+
+    # place files to handle daisy chain sleep issue
```

Comment:
What is the daisy chain sleep issue? Explain it in a comment

Reply:
Done in the commit below.